src/main.rs

```diff
@@ -270,9 +270,8 @@ fn main() {
         }
 
         "new" => {
-            // TODO: 'cobalt new nonsense' + unwrap = crash so provide default again
-            let filetype = matches.value_of("FILETYPE").unwrap_or("post");
-            let filename = matches.value_of("FILENAME").unwrap_or("new_post.md");
+            let filetype = matches.value_of("FILETYPE").unwrap();
+            let filename = matches.value_of("FILENAME").unwrap();
```

Comment:
As I mentioned before, you could make these "expects".  That way we document in the message why we think its safe to unwrap.